ReactWindows/ReactNative.Net46/Touch/TouchHandler.cs

```diff
@@ -0,0 +1,358 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Events;
+using System;
+using System.Collections.Generic;
+using System.Windows;
+using System.Windows.Input;
+using System.Windows.Media;
+
+namespace ReactNative.Touch
+{
+    class TouchHandler : IDisposable
+    {
+        private readonly FrameworkElement _view;
+        private readonly List<ReactPointer> _pointers;
+
+        private uint _pointerIDs;
+
+        public TouchHandler(FrameworkElement view)
+        {
+            _view = view;
+            _pointers = new List<ReactPointer>();
+
+            _view.MouseDown += OnPointerPressed;
+            _view.MouseMove += OnPointerMoved;
+            _view.MouseUp += OnPointerReleased;
+        }
+
+        public void Dispose()
+        {
+            _view.MouseDown -= OnPointerPressed;
+            _view.MouseMove -= OnPointerMoved;
+            _view.MouseUp -= OnPointerReleased;
+        }
+
+        private void OnPointerPressed(object sender, MouseButtonEventArgs e)
+        {
+            var originalSource = e.OriginalSource as DependencyObject;
+            var rootPoint = e.GetPosition(_view);
+            var reactView = GetReactViewTarget(originalSource, rootPoint);
+            if (reactView != null && _view.CaptureMouse())
+            {
+                var viewPoint = e.GetPosition(reactView);
+                var reactTag = reactView.GetReactCompoundView().GetReactTagAtPoint(reactView, viewPoint);
+                var pointer = new ReactPointer();
+                pointer.Target = reactTag;
+                pointer.PointerId = (uint)e.Device.GetHashCode();
+                pointer.Identifier = ++_pointerIDs;
+                pointer.PointerType = "M";
+                pointer.IsLeftButton = e.LeftButton == MouseButtonState.Pressed;
+                pointer.IsRightButton = e.RightButton == MouseButtonState.Pressed;
+                pointer.IsMiddleButton = e.MiddleButton == MouseButtonState.Pressed;
+                pointer.IsHorizontalMouseWheel = false;
+                pointer.IsEraser = false;
```

Comment:
You can delete this property if it doesn't make sense for WPF (along with IsEraser). They are not serialized in the mouse event payload by default.